docs/development/release.md

```diff
@@ -323,6 +323,14 @@ If this is a new minor version branch, then you will need to create a new Bamboo
 
 Bamboo will build and run lint and unit tests against that tagged release, publish the new packages to NPM, and then run the integration tests using those newly released packages.
 
+:::caution important
+
+  If this is a backport release, you **must** run the `master` branch in Bamboo immediately after the backport release is published. This is because the release process will overwrite the Cumulus documentation with the backport branch and will remove current versions. Running `master` will restore the current doc versions.
```

Comment:
This is a good backstop, but it's *much* quicker to revert the doc updates commit and push.  We should discuss this further.

Reply:
Yeah what do you think about this: https://github.com/nasa/cumulus/pull/3640/commits/f9a12b77769a383e1af363139bd795ecc2a93425

I kept "running main" as an option but inserted the quicker "revert the pages commit" before.